<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mDialog</class>
 <widget class="QDialog" name="mDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>296</width>
    <height>106</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Mâ–¼</string>
  </property>
  <widget class="QTextBrowser" name="textBrowser">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>2</y>
     <width>256</width>
     <height>71</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>80</y>
     <width>286</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>18</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="mcButton">
      <property name="text">
       <string>MC</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="mPlusButton">
      <property name="text">
       <string>M+</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="mRedButton">
      <property name="text">
       <string>M-</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
